eox_nelp/third_party_auth/tests/test_pipeline.py

```diff
@@ -60,17 +63,21 @@ def test_staff_user_raise_exc(self, user_kwargs):
         """Test the pipeline method try to match with uid, the user with matched username exists
         but is staff or superuser.
         Expected behavior:
-            - Raise EoxNelpAuthException exception
+            - The pipe return an HttpResponseForbidden object.
```

Comment:
```suggestion
            - The pipeline returns an HttpResponseForbidden object.
```

Reply:
Here I have some doubts I mean pipeline is not the list of pipes ?
and a pipe is only for one related to a function?
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/2131e07b-e4fb-48b2-a839-991f76219dd5)

Reply:
makes sense, however it's the first time that I see that someone uses `pipe` to refer to a single process, that make s me think about the Unix pipe symbol |

Reply:
I thing that could be used in that way. The unix pipe operator is used to connect, but here the pipes also changes the process in the interconnection xD.
_A pipe is a method used to pass information from one program process to another. Unlike other types of interprocess communication, a pipe only offers one-way communication by passing a parameter or output from one process to another._
https://www.techopedia.com/definition/3410/pipe
https://www.geeksforgeeks.org/pipe-system-call/

Reply:
were you able to find any reference where someone call this pipe ? I think this make sense but that looks like a literal translation that nobody use

Reply:
anyway I don't want to get into semantic discussions IMO pipe is not a common term and I prefer something like pipeline method or function but it's up to you